examples/game_vblank_draw.rs

```diff
@@ -1,7 +1,7 @@
 #![no_std]
 #![no_main]
 
-use gba::prelude::*;
+use gba::{prelude::*, RUST_IRQ_HANDLER};
```

Comment:
It looks like `RUST_IRQ_HANDLER` is exported as part of the prelude, so I don't think the direct import is necessary here (or in the other examples that use it: `hello.rs` and `mode3_pong_example_game.rs`.

Reply:
yeah i think i fixed the build errors at one port with RA adding the specific import, then later added it to the prelude.